testem.js

```diff
@@ -18,8 +18,7 @@ let config = {
       "args": [
         '--disable-gpu',
         '--headless',
-        '--remote-debugging-port=9222',
-        '--window-size=1440,900'
+        '--remote-debugging-port=9222'
```

Comment:
@chadhietala why did you change this?

Reply:
Doesn't seem required to have the window size for headless tests.

Reply:
Depends on what you're testing. For ember apps it's the default because otherwise media queries won't work properly because the window size is 0x0 px by default.